examples/cluster/plot_dbscan.py

```diff
@@ -34,8 +34,10 @@
 
 # Number of clusters in labels, ignoring noise if present.
 n_clusters_ = len(set(labels)) - (1 if -1 in labels else 0)
+n_noise_ = list(labels).count(-1)
```

Comment:
The numpy idiom might rather be `(labels == -1).sum()`. But I suppose I'm fine either way